### PR TITLE
Update requirements.txt + README.md of SD

### PR DESCRIPTION
**Pull request by Abhishek-Varma**
-- This commit includes two python modules as part of requirements.txt. 
-- It also updates README.md to also inclue `--no-use_tuned` for users to
   be able to try `hf_model_id` or `ckpt_loc` without any issue.

Signed-off-by: Abhishek Varma <abhishek@nod-labs.com>

### CODE REVIEW

**Review by powderluv — APPROVED** [review]

File: shark/examples/shark_inference/stable_diffusion/README.md [hunk 1]
```diff
@@ -14,19 +14,19 @@ Currently we support fine-tuned versions of Stable Diffusion such as:
...
 ```shell
-python .\shark\examples\shark_inference\stable_diffusion\main.py --hf_model_id="Linaqruf/anything-v3.0" --max_length=77 --prompt="1girl, brown hair, green eyes, colorful, autumn, cumulonimbus clouds, lighting, blue sky, falling leaves, garden"
+python .\shark\examples\shark_inference\stable_diffusion\main.py --hf_model_id="Linaqruf/anything-v3.0" --max_length=77 --prompt="1girl, brown hair, green eyes, colorful, autumn, cumulonimbus clouds, lighting, blue sky, falling leaves, garden" --no-use_tuned
 ```
 
 ## Run a custom model using a HuggingFace `.ckpt` file:
 * Install the following by running :-
 ```shell
-pip install omegaconf safetensors pytorch_lightning
+pip install pytorch_lightning
```
Comment (powderluv) [inline shark/examples/shark_inference/stable_diffusion/README.md:23]: you should call out the model that requires this.

**Review by powderluv — APPROVED** [review]

File: requirements.txt [hunk 2]
```diff
@@ -21,6 +21,8 @@ scipy
 ftfy
 gradio
 altair
+omegaconf
+safetensors
```
Comment (powderluv) [inline requirements.txt:25]: you will also have to add it to the spec files for web and main.py

**Review by powderluv — COMMENTED** [review]
We also need to add to the spec file in the same directory as main.py.

Also once we have @yzhang93  fix can we re-enable tuned ?

**Comment by Abhishek-Varma** [comment]
> We also need to add to the spec file in the same directory as main.py.
> 
> Also once we have @yzhang93 fix can we re-enable tuned ?

Hi @powderluv I've updated the spec file in both the places now :-
1. `shark/examples/shark_inference/stable_diffusion/shark_sd_cli.spec`
2. `web/shark_sd.spec`

Also, tested the following command on Windows Navi3 (6007) with the [latest fix](https://github.com/nod-ai/SHARK/pull/880) by @yzhang93 , but since the fix is CUDA specific we see ourselves in the same situation where we've to resort to using `--no-use_tuned` when `device=vulkan` :-

1. Initial command (Same command tried by one of the users who has raised an issue. Issue no. 886) : Didn't work.
```
python shark/examples/shark_inference/stable_diffusion/main.py --import_mlir --hf_model_id="andite/anything-v4.0" --max_length=77 --prompt="1girl, brown hair, green eyes, colorful, autumn, cumulonimbus clouds, lighting, blue sky, falling leaves, garden"
```

2. Same command: Worked when using `--no-use_tuned`.
```
python shark/examples/shark_inference/stable_diffusion/main.py --import_mlir --hf_model_id="andite/anything-v4.0" --max_length=77 --prompt="1girl, brown hair, green eyes, colorful, autumn, cumulonimbus clouds, lighting, blue sky, falling leaves, garden" --no-use_tuned
```

By default `device` is set `vulkan`, so in the above command you'd not see the `device` flag explicitly.
So, I tried another test, with `device` being set explicitly to `vulkan` and a new HuggingFace repo-id and saw similar pattern.

1. Initial command: Didn't work.
```
python shark/examples/shark_inference/stable_diffusion/main.py --precision=fp16 --device=vulkan --prompt="tajmahal, oil on canvas, sunflowers, 4k, uhd" --max_length=64 --import_mlir --hf_model_id="runwayml/stable-diffusion-v1-5"
```

2. Same command: Worked when using `--no-use_tuned`.
```
python shark/examples/shark_inference/stable_diffusion/main.py --precision=fp16 --device=vulkan --prompt="tajmahal, oil on canvas, sunflowers, 4k, uhd" --max_length=64 --import_mlir --hf_model_id="runwayml/stable-diffusion-v1-5" --no-use_tuned
```

So, my verdict would be to use `--no-use_tuned` flag for now while we have a fix for on-the-fly annotation for SD.

**Comment by powderluv** [comment]
Can you please respond to the folks reporting issues and on discord.

**Pull request merged**